src/pages/MoviesPage/MoviePage.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import moviepicsum from '../../assets/moviepicsum.jpeg'
+import { Footer, HCard, Navbar } from '../../components'
+import './MoviePage.css'
+
+export const MoviePage = () => {
+  return (
+    <div>
+      <Navbar />
+      <div className='movie-container'>
+        <h2 className='movie-title'>Movies</h2>
+        <div className='movie-card-container'>
+          <HCard
+            thumbnail={moviepicsum}
```

Comment:
thumbnail you don't need to  need to add your own image, 
can be done by inspecting youtube thumbail

Reply:
also title={' '}  time={' '} this need not to be done manually, it can be taken from data.js

Reply:
yes 👍